pkg/models/resources/v1alpha3/volumesnapshotclass/volumesnapshotclass.go

```diff
@@ -86,3 +95,18 @@ func (v *volumeSnapshotClassGetter) filter(object runtime.Object, filter query.F
 		return v1alpha3.DefaultObjectMetaFilter(snapshotClass.ObjectMeta, filter)
 	}
 }
+
+func (v *volumeSnapshotClassGetter) countVolumeSnapshots(name string) int {
+	snapshots, err := v.informers.Snapshot().V1().VolumeSnapshots().Lister().VolumeSnapshots("").List(labels.Everything())
```

Comment:
is this correct? shouldn't be `v.informers.Snapshot().V1().VolumeSnapshots().Lister().List(labels.Everything())`?

Reply:
My mistake, it has been changed.